packages/backend/src/controllers/authentication.ts

```diff
@@ -164,18 +168,27 @@ export const googlePassportStrategy: GoogleStrategy | undefined = !(
               ).href,
               passReqToCallback: true,
           },
-          async (req, issuer, profile, done) => {
+          async (
+              req: Express.Request,
+              accessToken: string,
+              refreshToken: string,
+              params: GoogleCallbackParameters,
+              profile: Profile,
+              done: VerifyCallback,
+          ) => {
               try {
+                  const issuer = 'https://accounts.google.com';
                   const { inviteCode } = req.session.oauth || {};
                   req.session.oauth = {};
-                  const [{ value: email }] = profile.emails;
+                  const [{ value: email }] = profile.emails || [];
                   const { id: subject } = profile;
                   if (!(email && subject)) {
-                      return done(null, false, {
+                      return done(null, undefined, {
                           message: 'Could not parse authentication token',
                       });
                   }
-                  // TODO why we ned this
+
+                  const { scope } = params;
```

Comment:
Should we store this as is? Apparently it should be a space-separated list of scopes, so we could parse it with `.split(' ')` - do we actually use the scopes?